client/account-recovery/reset-password/reset-password-form/test/index.jsx

```diff
@@ -78,24 +72,24 @@ describe( 'ResetPasswordForm', () => {
 			const wrapper = mount(
 				<ResetPasswordFormComponent
 					resetOptions={ exampleResetOptions }
+					pickedMethod={ null }
+					translate={ identity }
 				/>
 			);
 
 			// Expect the button to be disabled
 			expect( wrapper.find( '.reset-password-form__submit-button' ).prop( 'disabled' ) ).to.be.ok;
 		} );
 
-		it( 'should be disabled when clicked', function() {
+		it( 'should be disabled when isRequesting is on', function() {
```

Comment:
Oops. I think this was the 2nd time I did this. Updated by d054139681a6a4d6b558e004df8a6ea01d0fd6d5.